vocabularies/UI.xml

```diff
@@ -1448,6 +1448,11 @@ or `204 No Content` if the request included a `Prefer` header with a value of `r
         </Annotation>
       </Term>
 
+      <Term Name="AdaptationHidden" Type="Core.Tag" Nullable="false" AppliesTo="Property EntitySet EntityType">
```

Comment:
```suggestion
      <Term Name="AdaptationHidden" Type="Core.Tag" Nullable="false" AppliesTo="Property EntitySet EntityType">
        <Annotation Term="Common.Experimental" />
```

Do we want to start as experimental?

Reply:
If we want to use this as a "tagging term" and annotate with `<Annotation Term="UI.AdaptationHidden" />`, then we need a `DefaultValue="true"`.